### DISCUSSION TITLE
OG-Core Developer Meeting, Apr. 10, noon-1pm ET

### DISCUSSION THREAD

**Discussion by rickecon** [PSLmodels/OG-Core#865] [Meeting]
**Agenda for OG-Core developer meeting: 4/10/23**
12:00-1:00pm ET

Join Google Meet video conference
https://meet.google.com/ynf-zpdr-pbx

Join by phone
‪(US) +1 252-394-8749‬ PIN: ‪321 186 757‬#

This meeting is open to the public.

**Agenda**
* 2D smoothing spline in OG-Core, PR #839 by @prrathi 
* OG-USA [PR #69](https://github.com/PSLmodels/OG-USA/pull/69) test failures

Next meeting: Monday, April 24, noon-1pm ET

@rickecon @jdebacker @jpycroft @nikhilwoodruff @SeaCelo @sarvonz @prrathi

**Comment by nikhilwoodruff** [comment]
Won't be able to make this one unfortunately but no major updates from my side (@jdebacker I saw your issue on the UK synthetic data- that just needs me to store a microdata access token in the OG-UK repo, will get on it this week)
